tests/helpers/TestBaseResourceChecker.py

```diff
@@ -56,24 +58,24 @@ def test_cleanResources_found(self):
         self.kubernetes_service.getMongoObject.return_value = self.cluster_object
         self.checker.listResources = MagicMock(return_value=[self.cluster_object])
         self.checker.cleanResources()
-        self.assertEqual([call.getMongoObject('mongo-cluster', self.cluster_object.metadata.namespace)],
+        self.assertEqual([call.getMongoObject("mongo-cluster", "mongo-operator-cluster")],
```

Comment:
Why do we want to hardcode this value?

Reply:
If the variable value changes by accident, we would never find it in the test otherwise.